mapie/tests/test_regression.py

```diff
@@ -19,14 +19,19 @@
 
 from mapie._typing import ArrayLike, NDArray
 from mapie.aggregation_functions import aggregate_all
+from mapie.conformity_scores import (
+    ConformityScore, AbsoluteConformityScore, GammaConformityScore
+)
 from mapie.metrics import regression_coverage_score
 from mapie.regression import MapieRegressor
 from mapie.subsample import Subsample
 
 
 X_toy = np.array([0, 1, 2, 3, 4, 5]).reshape(-1, 1)
 y_toy = np.array([5, 7, 9, 11, 13, 15])
-X, y = make_regression(n_samples=500, n_features=10, noise=1.0, random_state=1)
+X, y = make_regression(
```

Comment:
Okay, maybe it would be interesting to keep an example where there are negative y values for other tests in `MapieRegressor` as it may/could be the cause for other issues in the future?

Reply:
Agreed, I added the bias for y nly in the last test.